README.md

```diff
@@ -150,6 +151,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [All IT eBooks](http://www.allitebooks.com/)
 * [B-OK](http://b-ok.org/)
 * [forcoder.org](http://forcoder.org/) IT ebooks and e-Learning videos for download
+* [LectuEpubGratis](https://lectuepubgratis.com/)
```

Comment:
I had a hard time finding CS/IT books. I'd go for not including this website tbh 😞 

```suggestion
```